src/components/App.tsx

```diff
@@ -5876,6 +5877,12 @@ class App extends React.Component<AppProps, AppState> {
       this.actionManager.getAppState(),
     );
 
+    const mayBeAllowToggleLineEditing =
+      actionToggleLineEditor.contextItemPredicate(
+        this.actionManager.getElementsIncludingDeleted(),
```

Comment:
shouldn't we be passing only non deleted elements for all actions ?

Reply:
Perhaps. Some actions (history) may need both, but they would be able to get them through `app` we could potentially pass as well.

Reply:
got it but for this action should I pass only non deleted ones ?

Reply:
we'd have to change the type which means for all actions?

Reply:
right, ok I will leave as is in that case